README.md

```diff
@@ -133,8 +133,6 @@ After installing tkn, create a link as kubectl-tkn
 $ ln -s /usr/local/bin/tkn /usr/local/bin/kubectl-tkn
   ```
 
-`kubectl` will find any binary named `kubectl-*` on your `PATH` and consider it as a plugin
```

Comment:
We might want something here though, such as:

Run the following to confirm `tkn` is available as a plugin:

Reply:
Thanks - applied these changes.